src/nameserver/name_server_impl.cc

```diff
@@ -9596,15 +9596,15 @@ std::shared_ptr<TabletInfo> NameServerImpl::GetTablet(const std::string& endpoin
 void NameServerImpl::CreateDatabaseOrExit(const std::string& db) {
     auto status = CreateDatabase(db, true);
     if (!status.OK() && status.code != ::openmldb::base::ReturnCode::kDatabaseAlreadyExists) {
-        LOG(FATAL) << "create database failed. code=" << status.GetCode() << ", msg=" << status.GetMsg();
+        LOG(ERROR) << "create database failed. code=" << status.GetCode() << ", msg=" << status.GetMsg();
```

Comment:
WHAT's the difference ? you still calls exit afterwards

Reply:
#3432